### PR TITLE
feat: #28 adding users table

### PR DESCRIPTION
**Pull request by aleku399**
you have to first type some thing in the input and delete it to show the data rows i.e type driving and delete it

### CODE REVIEW

File: src/views/components/UserTable/index.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      data = data.filter(row => {
+        return (
+          row.userName.includes(this.state.search) ||
+          row.role.includes(this.state.search)
+        );
+      });
+    }
+    return (
+      <div>
+        Search: <Input value={this.state.search} onChange={this.Search} />
```
Comment (epicallan) [inline src/views/components/UserTable/index.tsx]: why don't you use inbuilt filtering from `reactTable`?

File: src/views/components/UserTable/index.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,211 @@
+import { storiesOf } from "@storybook/react";
+import React from "react";
+import UserTable from "./index";
+
+storiesOf("components/UserTable", module).add("default", () => (
+  <UserTable
+    data={[
```
Comment (epicallan) [inline src/views/components/UserTable/index.stories.tsx]: extract `data` into  a `const` variable.

File: src/views/components/UserTable/index.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  data: User[];
+}
+
+const columns = [
+  {
+    Header: "Name",
+    columns: [
+      {
+        Header: "User Name",
+        accessor: "userName"
```
Comment (epicallan) [inline src/views/components/UserTable/index.tsx]: `userNames` should be clickable links that lead to user profiles.
see https://github.com/zeit/next.js#with-link & https://github.com/tannerlinsley/react-table/tree/v6#example

File: src/views/components/UserTable/index.tsx (hunk 1 above)
Comment (aleku399) [reply src/views/components/UserTable/index.tsx]: It will put a filter input  on each table and we end up with with two inputs

Comment (epicallan) [reply src/views/components/UserTable/index.tsx]: I believe it should be possible to set filtering on individual columns. See column props https://github.com/tannerlinsley/react-table/tree/v6#example
Screen Shot 2019-04-01 at 6 12 32 PM

**Pull request merged**